File: public/docs/_examples/displaying-data/dart/lib/app_component_1.dart
// #docregion
import 'package:angular2/core.dart';

@Component(
    selector: 'my-app',
    // #docregion template
    template: '''
      <h1>{{title}}</h1>
      <h2>My favorite hero is: {{myHero}}</h2>
    '''
    // #enddocregion template
    )
class AppComponent {
  String title = 'Tour of Heroes';
  String myHero = 'Windstorm';
}

File: public/docs/_examples/router/ts/app/app.component.1.ts
/* First version */
// #docplaster
// #docregion
import { Component } from '@angular/core';
// #docregion import-router
import { ROUTER_DIRECTIVES } from '@angular/router';
// #enddocregion import-router

@Component({
  selector: 'my-app',
  // #docregion template
  template: `
    <h1>Component Router</h1>
    <nav>
      <a [routerLink]="['/crisis-center']">Crisis Center</a>
      <a [routerLink]="['/heroes']">Heroes</a>
    </nav>
    <router-outlet></router-outlet>
  `,
  // #enddocregion template
  // #docregion directives
  directives: [ROUTER_DIRECTIVES]
  // #enddocregion directives
})

export class AppComponent { }
